gltf/cli.py

```diff
@@ -15,7 +21,7 @@ def main():
     parser.add_argument(
         '--version',
         action='version',
-        version='%(prog)s {}'.format(__version__)
+        version='%(prog)s {}'.format(metadata.version('panda3d-gltf'))
```

Comment:
This seems like the kind of thing that causes issues when using a deployment system, has this been tested to work in a deployed environment?

Reply:
I tested it by building it (in both 3.6 and 3.8), installing the wheel in a clean venv, and calling `gltf2bam --version`.  Did you mean a deployed, built, panda application?  Something else?